core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -438,16 +439,21 @@ public synchronized RaftGroup getCurrentGroup() {
   }
 
   private RaftClient createClient() {
```

Comment:
Is this still used?

Reply:
I think they should all use the same timeouts. Otherwise further investigations would be messy.